addons/product/views/product_attribute_views.xml

```diff
@@ -94,7 +94,10 @@
                 <field name="attribute_id" optional="hide"/>
                 <field name="name"/>
                 <field name="display_type" optional="hide"/>
-                <field name="html_color" attrs="{'invisible': [('display_type', '!=', 'color')]}" widget="color"/>
+                <field name="html_color"
+                       widget="color"
+                       readonly="1"
```

Comment:
Yes it is. Otherwise the color palette shows up while it is not editable, i.e the change on the color would not be recorded. The main objective of the task is to prevent this palette to appear here. Unless there is something to change in the widget, I would keep this reandonly.

Reply:
If the field was not always readonly, it could make sense, but this field here is never editable, be it in list/form/whatever view, because the field itself is readonly here on the model (cause related fields are readonly by default).